src/couch/src/couch_server.erl

```diff
@@ -1011,6 +1011,21 @@ names() ->
     N = couch_server:num_servers(),
     [couch_server:couch_server(I) || I <- lists:seq(1, N)].
 
+%% Try to lock an entry, must be unlocked at the time.
+try_lock(Table, DbName) ->
```

Comment:
Might be a good idea to assert the types, that `Table` is an atom and `DbName` a binary?

Reply:
have done this, good thought.